tools/plugin-release

```diff
@@ -445,6 +445,8 @@ def prepare(rel_name, archive):
     plugin.extractall(path=src_dir)
     plugin.close()
 
+    os.remove(archive)
```

Comment:
The archive is recreated latr. Removing the file seems to be preferable as we want to force its format later.